src/OnlineRetailPortal.Core/Translators/GetCategoryCoreResponseTranslator.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using System.Text;
+
+namespace OnlineRetailPortal.Core
+
+{
+    public static class GetCategoryCoreResponseTranslator
```

Comment:
check name?

Reply:
should i change this to CategoryCoreResponseTranslator